src/tests/directives.md

```diff
@@ -145,6 +145,8 @@ Some examples of `X` in `ignore-X` or `only-X`:
 - When particular debugger versions are matched: `ignore-gdb-version`
 - Specific [compare modes]: `compare-mode-polonius`, `compare-mode-chalk`,
   `compare-mode-split-dwarf`, `compare-mode-split-dwarf-single`
+- The two different test modes used by coverage tests:
+  `ignore-coverage-map`, `ignore-coverage-run`
```

Comment:
The style here is a bit inconsistent with most of the other list items, which omit the `ignore-` or `only-` prefix. I chose to use the full name here partly because it's more searchable, and partly because `only-coverage-map` etc are not in the permitted list of directives and therefore don't actually work.

Reply:
This is fine, thanks.